src/main/resources/payers/insurance_eligibilities.csv

```diff
@@ -1,8 +1,9 @@
-﻿Name,Poverty Multiplier,Income Threshold,Age Threshold,Qualifying Codes,Qualifying Attributes,Poverty Multiplier File,Spenddown File,Acceptance Likelihood,Sub-Eligibilities,Logical Operator
-StandardSocialSecurityDisability,,,,payers/eligibility_input_files/ssd_eligibility.csv,,,,0.9,,and
-StandardMedicare,,,65,,end_stage_renal_disease = true,,,,StandardSocialSecurityDisability,or
-StandardMedicaid,,,,,blindness = true,payers/eligibility_input_files/medicaid_income_eligibilities.csv,payers/eligibility_input_files/medicaid_mnil_eligibilities.csv,,,or
-StandardDualEligible,,,,,,,,,StandardMedicaid|StandardMedicare,and
-MedicaidPoverty2.0,2.0,,,,,,,,StandardMedicaid,or
-MedicareAge50,,,50,,,,,,StandardMedicare,or
-MedicareForAll*,,,0,,,,,,,
\ No newline at end of file
+﻿Name,Poverty Multiplier,Income Threshold,Age Threshold,Qualifying Codes,Qualifying Attributes,Poverty Multiplier File,Spenddown File,Acceptance Likelihood,Sub-Eligibilities,Logical Operator,Notes
+StandardSocialSecurityDisability,,,,payers/eligibility_input_files/ssd_eligibility.csv,,,,0.85,,and,
+MedicareEligible,,,65,,end_stage_renal_disease = true,,,,StandardSocialSecurityDisability,or,
+MedicaidEligible,,,,,blindness = true,payers/eligibility_input_files/medicaid_income_eligibilities.csv,payers/eligibility_input_files/medicaid_mnil_eligibilities.csv,,,or,
+DualEligible,,,,,,,,0.85,MedicareEligible|MedicaidEligible,and,

```

Comment:
```
DualEligible,,,,,,,,0.85,MedicareEligible|MedicaidEligible,and,
```
What is the effect of adding the `0.85` here?

85% of those who are dual eligible are actually accepted?

What happens to the other 15%?

Default to Medicaid or Medicare?

Reply:
Yup, 85% of eligible people will be accepted to Dual Eligible. The other 15% will be 50/50 randomly distributed to either Medicare or Medicaid.

This was part of a tuning effort to get enrollment rate closer to realistic distributions... If it doesn't make sense to include I can certainly remove it.

Reply:
I think that is OK. I want to understand the intent when I review the code that handles this.

Reply:
It might make more sense to default to Medicare if they are over-65, and otherwise randomly... That might introduce too much complexity though.

Reply:
Could do that by giving Medicare a higher priority than Medicaid. Right now, there's a 50/50 distribution since they're both the same priority level.